extensions/2.0/Khronos/KHR_texture_float_bc6h/README.md

```diff
@@ -0,0 +1,219 @@
+
+When the extension is used, it's allowed to use value `image/ktx2` for the `mimeType` property of images that are referenced by the `source` property of `KHR_texture_float_bc6h` texture extension object.
+
+At runtime, engines are expected to check hardware support for VK_FORMAT_BC6H_UFLOAT_BLOCK, if not present the texture should be transcoded converted to a suitable supported format.
+
+## glTF Schema Updates
+
+The `KHR_texture_float_bc6h` extension is added to the `textures` object and specifies a `source` property that points to the index of the `image` which defines a reference to the KTX v2 image in VK_FORMAT_BC6H_UFLOAT_BLOCK.
+
+The following glTF will load `image.ktx2` in clients that support this extension, and otherwise fall back to `image.png`.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_float_bc6h"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_float_bc6h": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.png"
+        },
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+When used in the glTF Binary (GLB) format the `image` that points to the KTX v2 resource uses the `mimeType` value of `image/ktx2`.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_float_bc6h"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_float_bc6h": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "mimeType": "image/png",
+            "bufferView": 1
+        },
+        {
+            "mimeType": "image/ktx2",
+            "bufferView": 2
+        }
+    ]
+}
+```
+
+### Using Without a Fallback
+
+To use KTX v2 image with Basis Universal supercompression without a fallback, define `KHR_texture_float_bc6h` in both `extensionsUsed` and `extensionsRequired`. The `texture` object will then have its `source` property omitted as shown below.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_float_bc6h"
+    ],
+    "extensionsRequired": [
+        "KHR_texture_float_bc6h"
+    ],
+    "textures": [
+        {
+            "extensions": {
+                "KHR_texture_float_bc6h": {
+                    "source": 0
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+### JSON Schema
+
+[texture.KHR_texture_float_bc6h.schema.json](schema/texture.KHR_texture_float_bc6h.schema.json)
+
+## KTX v2 Images with VK_FORMAT_BC6H_UFLOAT_BLOCK
+
+To cover usecases where a texture source shall have increased dynamic range
```

Comment:
Higher-fidelity material textures and non-prefiltered IBLs have different input data requirements and unlikely to be served by a single extension.

Reply:
So what is your suggestion?

Reply:
There were couple attempts at defining an IBL extension within the current ecosystem. My suggestion is to converge there. In parallel, we could work on WCG support in glTF. This is a big topic with major ecosystem implications, so it should start with a high-level discussion.

Reply:
Sure - I agree with that.
However, in the meantime I think we could make progress with either BC6H or RGB9E5 which are needed in order for us to define enviroment light extension.

Reply:
If the immediate goal of these extensions is to provide storage support for IBL, that's fine. Still, I wouldn't expect the finalized IBL extension to support more than one storage option.

Reply:
> If the immediate goal of these extensions is to provide storage support for IBL, that's fine. Still, I wouldn't expect the finalized IBL extension to support more than one storage option.

Well, the goal is to provide support for float source images (textures) - one immediate usage of this would be the IBL.  
Written the way it is it shall also be possible to use as a texture source for materials, ie the same way that KHR_texture_basisu is used.